apps/docs/pages/guides/api/rest/generating-types.mdx

```diff
@@ -2,7 +2,7 @@ import Layout from '~/layouts/DefaultGuideLayout'
 
 export const meta = {
   id: 'generating-types',
-  title: 'Generating Types',
+  title: 'Generating TypeScript Types',
```

Comment:
Should we just link to the reference page? There's quite a bit of duplicated content here

Reply:
We can keep the generating & GHA part here, but link to the reference on actually using the types?

Reply:
cc @RamiroND @MildTomato regarding SEO considerations here.